packages/@ember/-internals/glimmer/lib/modifiers/custom.ts

```diff
@@ -9,11 +11,24 @@ export interface CustomModifierDefinitionState<ModifierInstance> {
   delegate: ModifierManagerDelegate<ModifierInstance>;
 }
 
-export interface Capabilities {}
+export interface OptionalCapabilities {
+  disableLifecycleTracking?: boolean;
+}
+
+export interface Capabilities {
+  disableLifecycleTracking: boolean;
+}
 
 // Currently there are no capabilities for modifiers
-export function capabilities(_managerAPI: string, _optionalFeatures?: {}): Capabilities {
-  return {};
+export function capabilities(
+  managerAPI: '3.13',
```

Comment:
We need to allow whatever version introduced managers, and we should probably add a deprecation for custom managers not having their capabilities set (nearly all of the managers in existence that I can find do not set capabilities at all)